services/gateway/sources/version3/mappings/nft.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	id: '=,string',
+	nft: {
+		chainID: '=,string',
+		collectionID: '=,string',
+		index: '=,number',
+	},
+	owner: '=,string',
+	attributesArray: ['attributesArray', {
+		module: '=,string',
+		attributes: '=,string',
+	}],
+	lockingModule: '=,string',
+	isNftEscrowed: '=,boolean',
```

Comment:
Do we need this boolean value as `escrowedChainID` itself tells whether a NFT is escrowed or not?